csharp/ql/test/resources/stubs/_frameworks/Microsoft.AspNetCore.App/Microsoft.AspNetCore.Antiforgery.cs

```diff
@@ -6,7 +6,7 @@ namespace AspNetCore
     {
         namespace Antiforgery
         {
-            // Generated from `Microsoft.AspNetCore.Antiforgery.AntiforgeryOptions` in `Microsoft.AspNetCore.Antiforgery, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60`
+            // Generated from `Microsoft.AspNetCore.Antiforgery.AntiforgeryOptions` in `Microsoft.AspNetCore.Antiforgery, Version=7.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60`
```

Comment:
I wonder if we should remove the generation of this comment. Does it add any value? If needed, we could add a single comment at the top of the file mentioning which version of the assembly was used. And then all further diffs would be significantly smaller. What do you think?

Reply:
I agree, it would be good to have just a single comment at the beginning of the file.

Reply:
That is a good improvement both in terms of readability of changes going forward, but it also gave a significant reduction in the size of the stub (in terms of text).